Games/Stone_Paper_Scissor_Game.py

```diff
@@ -0,0 +1,100 @@
+import random
+
+print("""
+          ______      ______       ______ 
+         /  ___/     |  __  \     /  ___/
+        |  (__       | |__)  |   |  (__   
+         \__  \      |   ___/     \__  \  
+         ___)  | _   |  |    _    ___)  | _
+        /_____/ |_|  |__|   |_|  /_____/ |_| 
+""")
+
+print("""
+         Welcome to Stone Paper Scissor!
+""")
+
+print("""
+\n\t\t Instructions: \n\tThere will be 10 Matches to be played.
+""")
+
+
+winner = ""
+
+
+def Game():
+    userScore, compScore = 0, 0
+    a = 1
+    while a <= 10:
+        print("""
+________________________________________________________
+      """)
+        print(f"\n \t\tMatch : {a}\n")
+        user = input("Enter either 'Stone / Paper / Scissor' : ")
+        computer = random.randint(1, 100)
+        comp = ""
+
+        if computer <= 33:
+            comp = "Stone"
+        if computer > 33 and computer <= 66:
+            comp = "Paper"
+        if computer > 66 and computer <= 100:
+            comp = "Scissor"
+
+        print(f"""
+      Computer's choice :{comp}
+      User's choice :{user}
+      """)
+
+        # Computer
```

Comment:
You can use .lower() instead of using Stone and stone (or any of the three words) separately and the code can be more simplified.